### ISSUE TITLE
DateFormat.format.parseDate  missed Time ?

### ISSUE THREAD

**Issue by hafeyang** [phstc/jquery-dateFormat#75]
I use dateFormat.js without jquery plugin

```
DateFormat.format.parseDate("2015-01-16 12:55:54","yyyy-MM-dd HH:mm:ss")
/*returns object{
    date: Fri Jan 16 2015 00:00:00 GMT+0800 (中国标准时间) , // missing Time ? "00:00:00" ?
    time:{hour:"12",millis:"",minute:'55","second":"54",time:"12:55:54"}
}*/
```

**Comment by phstc** [comment]
Hey @hafeyang 

Could you submit a PR with your test example?

https://github.com/phstc/jquery-dateFormat/blob/master/test/issues_spec.js

Thanks

**Comment by phstc** [comment]
Closed by #76
